common/djangoapps/util/usermautic.py

```diff
@@ -0,0 +1,123 @@
+    "NL": "Netherlands", "NZ": "New Zealand", "NI": "Nicaragua",
+    "NE": "Niger", "NG": "Nigeria", "KP": "North Korea", "NO": "Norway",
+    "OM": "Oman", "PK": "Pakistan", "PW": "Palau", "PA": "Panama",
+    "PG": "Papua New Guinea", "PY": "Paraguay", "PE": "Peru",
+    "PH": "Philippines", "PL": "Poland", "PT": "Portugal",
+    "QA": "Qatar", "RO": "Romania", "RU": "Russia", "RW": "Rwanda",
+    "KN": "Saint Kitts and Nevis", "LC": "Saint Lucia", "WS": "Samoa",
+    "SM": "San Marino", "ST": "Sao Tome and Principe",
+    "SA": "Saudi Arabia", "SN": "Senegal", "RS": "Serbia",
+    "SC": "Seychelles", "SL": "Sierra Leone", "SG": "Singapore",
+    "SK": "Slovakia", "SI": "Slovenia", "SB": "Solomon Islands",
+    "SO": "Somalia", "ZA": "South Africa", "KR": "South Korea",
+    "SS": "South Sudan", "ES": "Spain", "LK": "Sri Lanka",
+    "VC": "Saint Vincent and the Grenadines", "PS": "Palestine",
+    "SD": "Sudan", "SR": "Suriname", "SZ": "Swaziland", "SE": "Sweden",
+    "CH": "Switzerland", "SY": "Syria", "TW": "Taiwan",
+    "TJ": "Tajikistan", "TZ": "Tanzania", "MK": "Macedonia",
+    "TH": "Thailand", "TG": "Togo", "TO": "Tonga",
+    "TT": "Trinidad and Tobago", "TN": "Tunisia", "TR": "Turkey",
+    "TM": "Turkmenistan", "TV": "Tuvalu", "GB": "United Kingdom",
+    "US": "United States", "UG": "Uganda",
+    "UA": "Ukraine", "AE": "United Arab Emirates", "UY": "Uruguay",
+    "UZ": "Uzbekistan", "VU": "Vanuatu", "VE": "Venezuela",
+    "VN": "Vietnam", "YE": "Yemen", "ZM": "Zambia", "ZW": "Zimbabwe"
+}
+
+
+def get_mautic_client():
+    if settings.FEATURES.get("MAUTIC_ENABLED"):
+        return MauticBasicAuthClient(
+            settings.MAUTIC_BASE,
+            username=settings.MAUTIC_USERNAME,
+            password=settings.MAUTIC_PASSWORD
+        )
+
+
+def register_mautic_contact(user_profile):
+    """
+    Creates or updates a user record in Mautic DB
+    :param user_profile:
+    :return:
+    """
+    client = get_mautic_client()
+    if not client:
+        return
+
+    user = user_profile.user
+    contacts = Contacts(client=client)
+
+    try:
+        resp = contacts.create({
+            'fullname': user_profile.name,
+            'username': user.username,
+            'firstname': user.first_name,
+            'lastname': user.last_name,
+            'verified': 1 if user.is_active else 0,
+            'email': user.email,
+            'yearofbirth': user_profile.year_of_birth,
+            'gender': user_profile.gender_display,
+            'levelofeducation': 
+                user_profile.level_of_education_display,
+            'city': user_profile.city,
+            'country': COUNTRIES.get(
+                user_profile.country.code, 'Unknown'),
+            'goals': user_profile.goals,
+        })
+    except Exception as e:
+        logger.error('Error submitting contact to Mautic')
+        logger.exception(e)
+        return
+
+    logger.info('User %s has been added successfully to Mautic',
```

Comment:
In English grammar, the last word carries the most empahasis. Thus it is stronger English to say `User %s has been added to Mautic successfully`